src/features/game/events/landExpansion/unlockFarmhand.ts

```diff
@@ -77,6 +77,15 @@ export const FARMHANDS: Equipped[] = [
     shirt: "Yellow Farmer Shirt",
     pants: "Farmer Overalls",
   },
+  {
+    background: "Farm Background",
+    body: "Goblin Potion",
+    hair: "Blondie",
+    shoes: "Black Farmer Boots",
+    tool: "Farmer Pitchfork",
+    shirt: "Yellow Farmer Shirt",
+    pants: "Farmer Overalls",
+  },
```

Comment:
These are supposed to be different each farm hand unlocked. There's a mixture of male/female goblin/bumpkin.

I think the last two should be changed so we don't get the same character three unlocks in a row.